app/views/posts/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>A single post</h1>
+<section class="mt-5">
+  <div class="container">
+  <div class="row border d-block m-0 mt-3">
+    <div class="border m-2">
+      <h5 class="m-2"><%= @post.title %> <span class="badge badge-secondary"><%= @post.created_at %></span></h5>
+      <p class="m-2"><%= @post.text %></p>
+      <%= link_to user_path(id: @post.author.id) do %>
+      <p class="m-2">by: <%= @post.author.name %></p>
+      <% end %>
+    </div>
+    <div class="border m-2">
+      <% @post.comments.each do |comment| %>
+        <p class="m-2 border-bottom"><%= comment.author.name %>: <%= comment.text %></p>
+      <% end %>
+    </div>
+  </div>
+</div>
+</section>
```

Comment:
- [ ] Kindly make this post show page to match the wireframe. As seen in the wireframe, the post author's name is at the top of the post beside the post title but yours is at the end. Kindly fix this
- [ ] Also add the number of comments and likes for the post as seen in the wireframe.

<img width="271" alt="chrome_eRPLqB9GVS" src="https://user-images.githubusercontent.com/81468675/163547361-86ad2f8f-d8e8-4a78-83ec-3520aba3d71d.png">